test/Microsoft.AspNet.Http.Abstractions.Tests/UseMiddlewareTest.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Builder.Internal;
+using Xunit;
+
+namespace Microsoft.AspNet.Http
+{
+    public class UseMiddlewareTest
+    {
+        [Fact]
+        public void UseMiddlewareNoParameters()
```

Comment:
A good unit test message says:
1. What it's testing
2. Under what condition
3. Expecting what result

E.g. `UseMiddleware_WithNoParameters_ThrowsException`